src/components/LoginPage/Slide3.js

```diff
@@ -0,0 +1,175 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import arrow from 'components/LoginPage/images/arrow.svg'
+import imageNoteContent from 'components/LoginPage/images/slide-3-note-content.svg';
+import imageMindmapExplanation from 'components/LoginPage/images/slide-3-mindmap-explanation.svg';
+import { Img, Caption } from 'components/LoginPage/slideComponents';
+import COLORS from 'components/LoginPage/colors';
+import { FONT_SIZE } from 'components/LoginPage/slideComponents';
+
+const Slide = styled.section`
+  padding-bottom: 0%;
+  text-align: center;
+
+  .note-example {
+    width: 61%;
+  }
+
+  .edit-example {
+    position: relative;
+    margin-top: -9%;
+  }
+
+  .hint-attach-files {
+    width: 22%;
+    padding-top: 3%;
+    padding-bottom: 3%;
+  }
+`;
+
+const HintContainer = styled.div`
+  margin-top: -65%;
```

Comment:
what is this `-65%` margin about?

Reply:
There are some changes in commit [03f0e80](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/03f0e8061689a6333fa330b9582c3322883b5d26)
I think it's must solved the problem